src/core/templates/buttons.hbs

```diff
@@ -11,7 +11,7 @@
     <div class="buttons-display">
         <div class="buttons-marking-icon icon display-none">
         </div>
-        <div class="buttons-display-inner">
+        <div class="buttons-display-inner" aria-live="rude">
```

Comment:
Hi @oliverfoster I have updated to use "assertive" as per the aria spec. I have also tested this out on IE11 with JAWS 18 as well as iOS on Safari as well as Chrome browser with ChromeVox and they all work fine. I personally think this is quite an improvement in terms of accessibility.